### ISSUE TITLE
Reading Effects chapter -> Could not match type ResponseFormat String with type AffjaxDriver

### ISSUE THREAD

**Issue by deemp** [purescript-halogen/purescript-halogen#799]
Hi! I'm going through the chapter [Performing Effects](https://github.com/purescript-halogen/purescript-halogen/blob/master/docs/guide/03-Performing-Effects.md#an-aff-example-http-requests) on my local machine and get an error with the latest version of `purescript-affjax`'s [get](https://pursuit.purescript.org/packages/purescript-affjax/13.0.0/docs/Affjax#v:get) function in this line 
```purs
response <- H.liftAff $ AX.get AXRF.string ("https://api.github.com/users/" <> username)
```
This breaking change is mentioned [here](https://github.com/purescript-contrib/purescript-affjax/blob/main/CHANGELOG.md#v1300---2022-04-28), so the guide should probably be updated to reflect this change.

Here's the updated [code](https://gist.github.com/br4ch1st0chr0n3/3f374a0bf52c8d69a5e1cb5ceb63d5f1). However, it doesn't run in try.purescript. Probably, due to not very fresh version of libraries there.

**Comment by JordanMartinez** [comment]
Just FYI. Try PureScript hasn't been updated to `0.15.0` yet.

**Comment by prikhi** [comment]
I ran into this recently, instead of explicitly passing `AW.driver`, I used the `get` function from `Affjax.Web`:

```purescript
response <- H.liftAff $ AW.get AXRF.string ("https://api.github.com/users/" <> username)
```

**Comment by yaitskov** [comment]
Yep. It is not clear where to get driver for get function from AX.

**Comment by garyb** [comment]
Closed by #815
